core/domain/param_domain.py

```diff
@@ -57,25 +87,49 @@ class ParamChange(object):
     """Value object for a parameter change."""
 
     def __init__(self, name, generator_id, customization_args):
-
+        """Initialze a ParamChange object with the specified arguments
+
+        Args:
+            name: unicode. The name of the parameter.
+            generator_id: unicode. The type of generator used to create the
+            parameter. eg. Copier
```

Comment:
Indent continuation lines by 4. Ditto below.

Pay attention to punctuation and capitalization. In this case: `... parameter, e.g., "Copier".`